src/vs/workbench/contrib/tasks/common/tasks.ts

```diff
@@ -1190,6 +1190,23 @@ export namespace KeyedTaskIdentifier {
 	}
 }
 
+export const enum TaskSettingId {
+	AutoDetect = 'task.autoDetect',
+	SaveBeforeRun = 'task.saveBeforeRun',
+	ShowDecorations = 'task.showDecorations'
+}
+
+export const enum TasksSettingId {
```

Comment:
I'm not sure what this is exactly, it is properties in the tasks schema?

Reply:
yes json schema v2 which is unfortunate bc don't think we can use these types as the keys there bc they're localized

Reply:
so maybe not worth doing this for those

Reply:
Yeah can't use in localize ids. How about naming this `TasksSchemaProperties`?